documentation/modules/ROOT/pages/modules/cdc-mysql-connector/r_mysql-supported-topologies.adoc

```diff
@@ -0,0 +1,39 @@
+// Metadata created by nebel
+//
+
+[id="mysql-supported-topologies_{context}"]
+= MySQL supported topologies
+
+The {prodname} MySQL connector supports the following MySQL topologies:
+
+Standalone::
+====
+When a single MySQL server is used, the server must have the binlog enabled (_and optionally GTIDs enabled_) so the {prodname} MySQL connector can monitor the server. This is often acceptable, since the binary log can also be used as an incremental link:https://dev.mysql.com/doc/refman/5.7/en/backup-methods.html[backup]. In this case, the MySQL connector always connects to and follows this standalone MySQL server instance.
+====
+
+Master and slave::
+====
+The {prodname} MySQL connector can follow one of the masters or one of the slaves (_if that slave has its binlog enabled_), but the connector only sees changes in the cluster that are visible to that server. Generally, this is not a problem except for the multi-master topologies.
+
+The connector records its position in the server’s binlog, which is different on each server in the cluster. Therefore, the connector will need to follow just one MySQL server instance. If that server fails, it must be restarted or recovered before the connector can continue.
+====
+
+High available clusters::
+====
+A variety of link:https://dev.mysql.com/doc/mysql-ha-scalability/en/[high availability] solutions exist for MySQL, and they make it far easier to tolerate and almost immediately recover from problems and failures. Most HA MySQL clusters use GTIDs so that slaves are able to keep track of all changes on any of the master.
+====
+
+Multi-master::
+====
+A link:https://dev.mysql.com/doc/refman/5.7/en/mysql-cluster-replication-multi-master.html[multi-master MySQL topology] uses one or more MySQL slaves that each replicate from multiple masters. This is a powerful way to aggregate the replication of multiple MySQL clusters, and requires using GTIDs.
```

Comment:
Would be good to replace the MySQL version with a variable globally and refer to that. It also should be 8.0 by now (or "stable", if they have some alias like that).